### ISSUE TITLE
Pink maps after generating many maps

### ISSUE THREAD

**Issue by capitalPSA** [tryashtar/image-map#138]
Hi. Thanks for putting this repo together. With this repo, I generated over 2,000 and 3,500 image maps! I found that after generating that many maps, there could be a limit on how many maps can be displayed at once. When the limit is reached, any additional loaded maps are pink.

See bug here:
https://bugs.mojang.com/browse/MCPE-152734

Have you experienced this? My first guess is that there is a limit set by Minecraft.

**Comment by tryashtar** [comment]
Bedrock being bedrock I'm afraid 🙃

**Comment by capitalPSA** [comment]
True. Thanks again for this repo. Would you like any support in maintaining or extending this repo?

**Comment by capitalPSA** [comment]
(I did not want to close this issue just yet. Sorry)

**Comment by tryashtar** [comment]
The application fulfills my needs and so I consider it complete and am spending my time elsewhere. There's definitely some wishlist items I won't get around to, but I wouldn't object to substantive PRs

**Comment by capitalPSA** [comment]
I think this issue has been fixed by Minecraft's dev team. 
As I was browsing the issues in the bedrock issue tracker, I came across a reference to "Remove Texture Limit" in the video settings. It must have been added around 2 months ago (July 2024?).

I tried it out now and, from a preliminary test, I think this option fixes my problem. There are no more pink tiles and pink UI.

Once I am satisfied with this fix, I will close my issue on their issue tracker.
